### ISSUE TITLE
Chrome Mosh app doesn't load `powerline-web-fonts`.

### ISSUE THREAD

**Issue by NHDaly** [wernight/powerline-web-fonts#9]
This is cross-referencing https://github.com/rpwoodbu/mosh-chrome/issues/145.

The powerline-web-fonts doesn't work in the chrome mosh app. It works correctly in [Secure Shell](https://chrome.google.com/webstore/detail/secure-shell/pnhechapfaindjhompbnflcldabbghjo), but not in [mosh](https://chrome.google.com/webstore/detail/mosh/ooiklbnjmhbcgemelgfhaeaocllobloj).

According to https://github.com/rpwoodbu/mosh-chrome/issues/145, it looks like mosh is complaining about a "Content Security Policy directive":

`Refused to load the stylesheet 'https://cdn.rawgit.com/wernight/powerline-web-fonts/8040cf32c146c7cd4f776c1484d23dc40685c1bc/PowerlineFonts.css' because it violates the following Content Security Policy directive: "style-src 'self' blob: filesystem: data: chrome-extension-resource: 'unsafe-inline'".`

Could this possibly be an issue with the fonts package? Does the css need to do something to make the security policy happy?

Thanks!

**Comment by wernight** [comment]
It looks more like a security limitation of that extension. I've seen another similar that only allows local resources (which is impossible on a Chromebook).

**Comment by NHDaly** [comment]
:) I just came across this package again via google searching years later when setting up my new chromebook, and just wanted to say "❤️ Thanks for an awesome package that Just Worked!" :) I followed the README instructions to install the fonts into Crosh, and it worked perfectly. Thanks!

Haha i came to the Issues page to see if there was a "Thank you" issue that i could +1, only to find my own post from years ago! 😂
